1208-projects/my_projects.feature

```diff
@@ -1,74 +1,78 @@
 
     In order to know of all project where I'm involved
-    I want to see lists of active and concluded project, and have access to manage those I'm responsible of
+    As a User
+    I want to see lists of active and concluded projects, and have access to manage those I'm responsible of
+
+    In order to keep projects I'm responsible of up to date
+    As a User
+    I want easy access to manage those 
 
     In order to see all details of a project
-    I want to be able to navigate to the landing page of a project
+    As a User
+    I want easy access to the landing page 
+
+    In order to locate the project in mind
+    As a User
+    I want easy access to search in My Projects
 
     In order to create new projects
-    I want to be able to create new projects
+    As a User
+    I want easy access to create new projects
 
-    In order to locate the project in mind
-    I want to be able to search in My Projects
+    Background: 
+        Given A User is logged in
 
     @2874a
+    @updated
     Scenario: User opens My Projects
-        When they navigate to the My Projects Page
-        Then they see a Create New Project Button
-        And they see a search field
-        And they see a list of Active Projects
-        And they see a collapsed list of Concluded Projects
-        And the lists contains Projects where the User has the role as:
+        When a User navigate to My Projects Page
+        Then the User see a option to create a new project
+        And the User see a search field
+        And the User see a list of Active Projects
+        And the User see a collapsed list of Concluded Projects
+        And the lists contains Projects where the User has one of the following roles:
             | Project Owner         |
             | Project Manager       |
             | Local Project Manager |
             | Participants          |
-        And each Project has:
-            | Title           |
-            | Institution     |
-            | Project Manager |
-        And each Project has an Edit button if the User has role: 
+        And each Project views it's:
+            | Title                    |
+            | Coordinating Institution |
+            | Project Manager          |
+        And each Project can be edited if the User has one of the following roles: 
             | Project Owner         |
             | Project Manager       |
             | Local Project Manager |
         And the list can be sorted by:
-            | Title           |
-            | Institution     |
-            | Project Manager |
-        And they see pagination buttons for the Concluded Projects list
+            | Title                    |
+            | Coordinating Institution |
+            | Project Manager          |
+        And the User can navigate through the possible long list of Concluded Projects
```

Comment:
litt usikker på hva dette betyr, men antar "navigate through" betyr at de skal kunne åpnes, så kanskje et eget sceanrio for dette?